Layout.py

```diff
@@ -2,16 +2,16 @@
 import json
 import sublime
 import sublime_plugin
-
+from time import sleep
 
 PLUGIN_NAME = 'Layout'
-PLUGIN_PATH = os.path.join(sublime.packages_path(), PLUGIN_NAME)
-LAYOUTS_PATH = os.path.join(PLUGIN_PATH, 'layouts')
 
 X_MIN, Y_MIN, X_MAX, Y_MAX = (0, 1, 2, 3)
 
 
 def plugin_loaded():
+    PLUGIN_PATH = os.path.join(sublime.packages_path(), PLUGIN_NAME)
```

Comment:
I keep getting errors for loading layouts after closing and reopening Sublime, if these os.path functions are in the global section, so I moved them in defs